tensorflow_encrypted/protocol/__init__.py

```diff
@@ -2,14 +2,17 @@
 
 # from unencrypted_native import UnencryptedNative
 # from unencrypted_fixedpoint import UnencryptedFixedpoint
-from .pond import Pond
+from .pond import Pond, TFEVariable, TFETensor
```

Comment:
the general types should be moved out of Pond

Reply:
I've added them back because I was running into circular dependencies.

If you create a `types` file to define Union of types like `TFEVariable` depending on classes that are inside of another file and the class have functions that you want to type with the types in the `type` file. 
Python is not happy with it.

This probably means that we too much things inside this `Pond` file.

Reply:
maybe we need to introduce some base tensor types that the pond classes inherit from

Reply:
#93